src/components/Badge/Indicator/Indicator.module.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../../styles/themes";
+
+$indicator-size: 8px;
+
+.indicator {
+  width: $indicator-size;
+  height: $indicator-size;
+  border-radius: 50%;
+
+  &.primary {
```

Comment:
Let's pass the right color enum to the counter / indiction component instead of override the css (override component styles by custom class names is less recommended practice, I can explain more about it if you want)

Reply:
not sure I got ya, can you please elaborate?

Reply:
talked about it :)